master-protected-branches/master.cfg

```diff
@@ -176,6 +176,17 @@ f_tarball.addStep(steps.Compile(command=["cmake","../server"], workdir='build/mk
 f_tarball.addStep(steps.Compile(command=["make", "dist"], workdir='build/mkdist', description="make dist"))
 f_tarball.addStep(steps.SetPropertyFromCommand(property="mariadb_version", command="basename mariadb-*.tar.gz .tar.gz", workdir="build/mkdist"))
 f_tarball.addStep(steps.SetPropertyFromCommand(property="master_branch", command=util.Interpolate("echo " + "%(prop:mariadb_version)s" + " | cut -d'-' -f 2 | cut -d'.' -f 1,2")))
+f_tarball.addStep(steps.ShellCommand(name="Check branch version", command=['bash', '-c',
+    util.Interpolate('''
+        master_branch='\<%(prop:master_branch)s\>'
+        if [[ "%(prop:branch)s" =~ $master_branch ]]; then
+            exit 0
+        else
+            echo "Package VERSION and branch name differs"
+            exit 1
```

Comment:
This is a bit curious, why not checking only the negative case, it will `exit 0` if check passes anyway.

```bash
if [[ ! "%(prop:branch)s" =~ $master_branch ]]; then
  echo "Package VERSION and branch name differs"
  exit 1
fi
```

Reply:
you are right, but I was thinking it's more clear to also have `exit 0`